packages/store-engine-dexie/src/index.test.ts

```diff
@@ -141,6 +141,31 @@ describe('IndexedDBEngine', () => {
     });
   });
 
+  describe('save', () => {
+    it('generates primary keys', async () => {
+      const TABLE_NAME = 'test-table';
+
+      engine = new IndexedDBEngine();
+      const db: Dexie = await engine.init('primary-key-store');
+      db.version(1).stores({
+        [TABLE_NAME]: '++primaryKey, testValue',
+      });
+      await db.open();
+
+      const entity = {
+        testValue: 'value',
+      };
+
+      type ExpectedResult = typeof entity & {primaryKey: number};
+
+      const primaryKey = await engine.updateOrCreate<number>(TABLE_NAME, undefined as any, entity);
```

Comment:
```suggestion
      const primaryKey = await engine.updateOrCreate<number | undefined>(TABLE_NAME, undefined, entity);
```

Can we do this to get rid off `any`?